scripts/ci/install-test-dependencies.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 set -e
 
-pip3 install wheel
-
-pip3 install fetchai-ledger-api==0.5.1
-pip3 install -i https://test.pypi.org/simple/ fetchai-netutils==0.0.6a1
-
-pip3 install pyyaml
+pip3 install --no-cache-dir --ignore-installed --force-reinstall wheel pyyaml requests
+# ???
+pip3 install --no-cache-dir --ignore-installed --force-reinstall git+https://github.com/wkaluza/ledger-api-py.git@wip_consolidate_smart_and_synergetic_contracts#egg=fetchai-ledger-api
+pip3 install --no-cache-dir --ignore-installed --force-reinstall --index-url https://test.pypi.org/simple/ fetchai-netutils==0.0.6a1
```

Comment:
I guess these will need to be updated before the merge

Reply:
Yes, the API will need to be reviewed and released before this can be merged. I created a PR; https://github.com/fetchai/ledger-api-py/pull/34